test/runtests.jl

```diff
@@ -1,355 +1,347 @@
-using ECOS, SCS, MathProgBase
-import JuMP
+
 using ConicBenchmarkUtilities
+using MathOptInterface
+MOI = MathOptInterface
+MOIU = MathOptInterface.Utilities
+MOIB = MOI.Bridges
+using GZip
+using Test
+using SCS
```

Comment:
There's not really a good reason that these tests need to depend on a solver. This is super indirect (it's testing f(X) = f(Y) where f() is solve with SCS). When a test fails, you'd have to do a lot of digging to see why the answer doesn't match what's expected. We have new tools in MOI that can help. I would recommend structuring tests as follows.

For reading CBF, use `MOIU.loadfromstring!` (e.g., https://github.com/JuliaOpt/JuMP.jl/blob/a8a46e5f9ff9eed7ad31548d69be76283f5f447f/test/generate_and_solve.jl#L238) to build an MOI model that should match what's written down in the CBF file. Then read the CBF file and use `MOIU.test_models_equal` to check that that the data matches.

For writing, same thing in the opposite direction.

Reply:
Yeah, here is how I do it in MOF: 
https://github.com/odow/MathOptFormat.jl/blob/21e79f41cd9a2eca38cc990437a6d21c95967e89/test/runtests.jl#L9-L16